### PR TITLE
Add support for .NET 6

### PR DESCRIPTION
**Pull request by steinmal**
Replace netcoreapp3.0 target with .net6.0-windows

### CODE REVIEW

File: Source/Examples/PropertyGrid/UndoRedoDemo/UndoRedoDemo.csproj [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <ProjectReference Include="..\..\..\PropertyTools\PropertyTools.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Caliburn.Micro" Version="3.2.0" />
-    <PackageReference Include="Expression.Blend.Sdk" Version="1.0.2" />
```
Comment (steinmal) [inline Source/Examples/PropertyGrid/UndoRedoDemo/UndoRedoDemo.csproj:14]: Seems to not be needed for .NET 6. The relevant parts are included.

File: Source/Examples/PropertyGrid/UndoRedoDemo/AppBootstrapper.cs [hunk 2]
```diff
@@ -18,7 +18,7 @@ public AppBootstrapper()
 
         protected override void OnStartup(object sender, StartupEventArgs e)
         {
-            this.DisplayRootViewFor<ShellViewModel>();
+            this.DisplayRootViewForAsync<ShellViewModel>();
```
Comment (objorke) [inline Source/Examples/PropertyGrid/UndoRedoDemo/AppBootstrapper.cs]: "await" this?

File: Source/Examples/PropertyGrid/UndoRedoDemo/ViewModels/ShellViewModel.cs [hunk 3]
```diff
@@ -217,7 +217,7 @@ public void DeleteItem()
         /// </summary>
         public void Exit()
         {
-            this.TryClose();
+            this.TryCloseAsync();
```
Comment (objorke) [inline Source/Examples/PropertyGrid/UndoRedoDemo/ViewModels/ShellViewModel.cs]: "await" this?

File: Source/PropertyTools.Wpf/PropertyTools.Wpf.csproj [hunk 4]
```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+﻿<Project Sdk="Microsoft.Net.Sdk">
   <PropertyGroup>
     <LanguageTargets>$(MSBuildToolsPath)\Microsoft.CSharp.targets</LanguageTargets>
-    <TargetFrameworks>net45;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks>net45;net6.0-windows</TargetFrameworks>
```
Comment (objorke) [inline Source/PropertyTools.Wpf/PropertyTools.Wpf.csproj]: maybe we should keep the `netcoreapp3.0` target?

File: README.md [hunk 5]
```diff
@@ -42,7 +42,7 @@
 ### Supported frameworks
 
 - Microsoft .NET 4.5.2 or later
-- Microsoft .NET Core 3.0
```
Comment (objorke) [inline README.md:45]: keep this?

File: Source/Examples/Controls/ControlDemos/ControlDemos.csproj [hunk 6]
```diff
@@ -1,6 +1,6 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+﻿<Project Sdk="Microsoft.Net.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>net6.0-windows</TargetFramework>
```
Comment (objorke) [inline Source/Examples/Controls/ControlDemos/ControlDemos.csproj]: change all demo projects to `net7.0-windows`?

**Comment by objorke** [comment]
#315

**Review by objorke — COMMENTED** [review]
Please consider these comments

**Comment by objorke** [comment]
Thanks! Looks good to me!

**Pull request merged**